lib/util/include/hse_util/base.h

```diff
@@ -47,9 +47,7 @@
 
 /* clang-format off */
 
-#define HSE_NAME "Heterogeneous-memory Storage Engine"
-
-#define STR_SAFE(_str)  ((_str) ?: "")
+#define STR_SAFE(_str)  ((_str) ? (_str) : "")
```

Comment:
Remove it, I don't see any use cases, and since it evaluates it's argument twice is just a disaster waiting to happen in some unimaginable use case :-)